packages/core/src/Spinner/Spinner.tsx

```diff
@@ -70,14 +78,61 @@ const propTypes = {
   ]),
 }
 
-const Spinner: React.FC<InferProps<typeof propTypes>> = ({ children, color, ...props }) => {
+const GradientRing: React.FC<InferProps<typeof propTypes>> = ({ strokeWidth = '6px', ...props }) => {
```

Comment:
Default stroke width changed to 6px from border-width of 2px. This was required to achieve similar visuals with a gradient